bin/conjur-cli.rb

```diff
@@ -78,6 +78,14 @@ def test_select
       exec "rake authn_local:run"
     end
 
+    # Start the rotation "watcher" in a separate thread
+    rotations_thread = Thread.new do
+      # exec "rake expiration:watch[#{account}]"
+      exec "rake expiration:watch"
```

Comment:
This is going to replace the current process, which is not what we want.

I suggestion going with the `fork / exec` model used for the other sub-tasks.